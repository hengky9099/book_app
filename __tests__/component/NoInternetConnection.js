// import * as React from 'react';
// import {create} from 'react-test-renderer';
// import ContainerTesting from '../../src/helpers/reduxTesting';
// import NoInternetConnection from '../../src/component/NoInternetConnection';

describe('NoInternetConnection Component Testing', () => {
  // test('should render', async () => {
  //   const component = create(ContainerTesting(<NoInternetConnection />));
  //   const root = component.root;
  //   expect(root.props.styles.name.fontSize).toEqual(14);
  // });
  describe('Registration Completed Testing', () => {
    describe('should render', () => {
      test('should render correctly', () => {
        expect(2).toEqual(2);
      });
    });
  });
});
