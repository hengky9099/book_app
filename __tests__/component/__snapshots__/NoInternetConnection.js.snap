// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoInternetConnection Component Testing Registration Completed Snapshot Snapshot 1`] = `
<View
  testID="NoInternetConnection"
>
  <Text
    style={
      Object {
        "fontSize": 37.714285714285715,
        "marginTop": 392.85714285714283,
        "textAlign": "center",
      }
    }
  >
    No Internet Connection, Check your internet
  </Text>
  <View
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#4D96FF",
        "borderRadius": 15.714285714285714,
        "height": 62.857142857142854,
        "justifyContent": "center",
        "marginLeft": 62.857142857142854,
        "marginTop": 471.42857142857144,
        "opacity": 1,
        "width": 408.57142857142856,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
        }
      }
    >
      Back to Login
    </Text>
  </View>
</View>
`;
